C/manual/ch_Reports.xml

```diff
@@ -3095,7 +3095,7 @@ Income (type INCOME)
                 </listitem>
 
                 <listitem>
-                  <para><guilabel>Report's currency</guilabel>: Select the currency to display the values in - see
+                  <para><guilabel>Report's currency</guilabel>: Select the currency to display the values in &minus; see
```

Comment:
Caution, this should be `—` (&mdash; or enter [AltGr]+[Shift]+[-]).
In this case it is obvisious by the following <q>see …</q>.

Reply:
Much better to use the entity: It's both easier to enter (key combinations are localized and OS-specific) and especially in the case of `&mdash;` (U+2014), `&ndash;` (U+2013), `&minus;`, (U+2011), and `&hyphen;` (U+2010, non-breaking U+2011) and the ASCII hyphen-dash (U+002d) that are hard to tell apart in source strings.

But I think it's better still to avoid using any of them except `&minus;` in its arithmetic meaning because we want the documentation translated. Correct usage of the various dashes and hyphen in English can be so obscure that many native English speakers don't even know that there's a difference (it's wrong throughout this PR: `&minus;` is for arithmetic and shouldn't be used in a sentence). Expecting non-native English speakers to correctly understand and translate them is asking a lot. em-dashes are usually a substitute for parentheses. That's the case here, so `(see <xref linkend="report-common"/>)` would be better.

Reply:
Interesting, @jralls . There seems to regional differences. In german books parentheses are used in formulas, but rarely in text.

We should try to find some consent and update the wiki section.

Reply:
Punctuation is very language specific. I wouldn't expect anything beyond full stops to transfer even between closely related languages like German, French, and English, never mind non-european ones. Even native speakers and writers of English have trouble with the finer points of punctuation, and there are lots of heated disputes about some practices, e.g. the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma). As it happens another hot dispute is how to set off a parenthetical expression. Sorry, I couldn't find a good reference for this, so here goes: The basic form that a lot of people think should be the only one is to use a pair of commas. The other inline choices are parentheses and em-dashes. One can also use footnotes or endnotes, an especially good idea if the aside will be lengthy.

The section in the wiki page about dashes just points to https://www.thepunctuationguide.com/index.html. I have no idea who wrote it but the sections are explanatory rather than style guidance. The flavor of English used by the original author doesn't seem important at least in that case.

Reply:
@jralls and @fellen I'm fine continuing however you want with this bit of it, but I should mention that the changes outside the Budget Balance Sheet were inadvertent. I'd forgotten to limit the scope of my search/replace.  Perhaps I should undo all these other changes?  I'm not sure I want to be on the hook for this entire xml file at this time.